src/components/user/ha-user-picker.ts

```diff
@@ -45,7 +45,9 @@ class HaEntityPicker extends LitElement {
           <paper-icon-item data-user-id="">
             No user
           </paper-icon-item>
-          ${this._sortedUsers(this.users).map(
+          ${this._sortedUsers(
+            this.users!.filter((user) => !user.system_generated)
```

Comment:
The filtering should be done in `memoizeOne`, so it is not done on every render but only when the users change